src/components/Loans/ClosedLoans/ClosedLoans.js

```diff
@@ -302,6 +330,10 @@ class ClosedLoans extends React.Component {
     const anonymizeString = <FormattedMessage id="ui-users.anonymize" />;
     const loansSorted = _.orderBy(loans,
       [this.sortMap[sortOrder[0]], this.sortMap[sortOrder[1]]], sortDirection);
+    const columnHeadersMap = this.columnHeadersMap;
+    const clonedLoans = JSON.parse(JSON.stringify(this.props.loans)); // Do not mutate the actual resource
+    const recordsToCSV = this.buildRecords(clonedLoans);
+    const onlyFields = columnHeadersMap;
```

Comment:
```
const columnHeadersMap = this.columnHeadersMap;
const onlyFields = columnHeadersMap;
```

seems like excess for me as `this.columnHeadersMap` can be passed straight away to the component.

```
<ExportCsv data={recordsToCSV} onlyFields={this.columnHeadersMap} />
```